addon/components/a11y-landmark.js

```diff
@@ -12,6 +12,28 @@ const LANDMARK_NAVIGATION_ROLE = {
     div: 'div'
 };
 
+const VALID_LANDMARK_ROLES = [
+    'banner',
+    'navigation',
+    'aside',
```

Comment:
@EndangeredMassa to clarify here- an "aside" is not a valid landmark role. It would need to be 'complementary' instead of 'aside' under VALID_LANDMARK_ROLES.